cmake/Helpers.cmake

```diff
@@ -131,9 +131,9 @@ endmacro()
 # Helper to print the configuration of a 2-valued or 3-valued option 'var'
 # with prefix 'str'.
 macro(print_config str var)
-  if("${var}" STREQUAL "ON")
+  if(${var} STREQUAL "ON")
     set(OPT_VAL_STR "on")
-  elseif("${var}" STREQUAL "OFF")
```

Comment:
I think we need to keep it as `"${var}"`, in case `${var}` is empty.

Reply:
Indeed!